### PR TITLE
[MIG] Migrate database_cleanup on 9.0

### PR DESCRIPTION
**Pull request by adrienpeiffer**

### CODE REVIEW

File: database_cleanup/__openerp__.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 {
     'name': 'Database cleanup',
-    'version': '8.0.0.1.0',
+    'version': '9.0.0.1.0',
```
Comment (pedrobaeza) [inline database_cleanup/__openerp__.py]: 9.0.1.0.0

**Comment by tremlin** [comment]
Code review okay, brief test in runbot :+1:

**Comment by astirpe** [comment]
Tested on v9, it works properly :+1: Code review also ok

**Comment by hbrunn** [comment]
@adrienpeiffer I migrated the code to the new api and did some improvements (like tests) in https://github.com/OCA/server-tools/pull/469, do you agree to close this one?

**Comment by moylop260** [comment]
@hbrunn 👍  use a PR with new api changes too https://github.com/OCA/server-tools/pull/469

**Comment by adrienpeiffer** [comment]
@hbrunn ok, no problem

**Pull request closed without merging**